packages/test/test-utils/src/localLoader.ts

```diff
@@ -64,3 +95,22 @@ export async function createAndAttachContainer(
 
 	return container;
 }
+
+/**
+ * Creates a detached Container and attaches it.
+ * @param source - The code details used to create the Container.
+ * @param loaderProps - The loaderProps to use to initialize the container.
+ * @param attachRequest - The request to create new from.
+ * @internal
+ */
+export async function createAndAttachContainerUsingLoaderProps(
```

Comment:
I assume this is getting the more verbose name because createAndAttachContainer is actually in use and can't be made `@internal`?  Otherwise the "recommended" approach should get the friendlier name.

Also, seems the existing createAndAttachContainer should get `@deprecated`?  It does next to nothing so IMO doesn't need a replacement, and it requires the now-deprecated `IHostLoader`.

Reply:
Shorten the name by elimination "Loader" as we are not using that anyway and also deprecated the createAndAttachContainer method